docs/blog/newsletter-september-2022.rst

```diff
@@ -0,0 +1,91 @@
+
+--------------------------------
+Collecting helpful user feedback
+--------------------------------
+
+Getting helpful user feedback is an eternal concern for documentarians. Is there a collection method that both encourages user participation and delivers categorizable data about docs quality and effectiveness? Open comment fields give users space to expand and offer details, but lengthy comments can be difficult to categorize for higher-level analysis.
+
+One-click ratings mechanisms with limited options offer a low obstacle for users---think thumbs up/thumbs down icons or a range of smiley faces like `HappyOrNot <https://www.happy-or-not.com/en/solution/>`__. The limited options are helpful for finding patterns in feedback and some documentation platforms even have such ratings tools built it. The drawback is that ratings scales often don't include a way for users to provide details, so you don't get any information about the reasons behind the user ratings. If you choose to collect ratings, take care to offer only a few options to keep the feedback meaningful.
+
+Some have had good results with prefilled feedback options as a middle ground between open-ended text fields and limited one-click rating scales. For example, you might ask users to complete a sentence from a list of short answers, such as completing "This page is..." with options like "helpful," "incomplete," "confusing," or "too long." The prefilled answers provide more detailed information that's still categorizable without burdening users. If desired, you can include options for sending detailed feedback or even creating tasks for doc team review.
+
+------------------------------------------------
+Improving communication about what's in releases
+------------------------------------------------
+
+We recently had a discussion about how to keep communication in sync between engineering, product managers (PMs), and documentation. In particular, about making sure that tech writers know if the product is changing so they know what needs doing for a release.
+
+The original poster hadn't had much success asking engineering to keep them informed -- often engineering would forget. So they were thinking of trying to improve this with better process, via PMs; surely PMs keep tabs on what's changing in the product and are aware of what changes might need doc work?
+
+The answer is, it depends, because how PMs work with engineering can vary hugely among organizations and even among teams. Some PMs are the main point of contact for what's changing, and want to be kept in the loop. Oothers are focused on handling user requests and reviewing feature designs and aren't really involved in launching and so engineering is a better source.
```

Comment:
```suggestion
The answer is, it depends, because how PMs work with engineering can vary hugely among organizations and even among teams. Some PMs are the main point of contact for what's changing, and want to be kept in the loop. Others are focused on handling user requests and reviewing feature designs and aren't really involved in launching and so engineering is a better source.
```

Reply:
This could probably use rewording

> thers are focused on handling user requests and reviewing feature designs and aren't really involved in launching and so engineering is a better source.